bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx

```diff
@@ -2,26 +2,49 @@ import React, { Component, PropTypes } from 'react';
 import styles from '../styles';
 import _ from 'underscore';
 import cx from 'classnames';
-
 import Icon from '/imports/ui/components/icon/component';
+import { defineMessages, injectIntl } from 'react-intl';
+
+const intlMessages = defineMessages({
+  exitfullscreenLabel: {
```

Comment:
Because this is a generic component, I don't think any code for specific implementations should be here, like this fullscreen stuff.